SwrveConversationSDK/src/main/java/com/swrve/sdk/SwrveConversationConstants.java

```diff
@@ -0,0 +1,9 @@
+package com.swrve.sdk;
+
+/**
+ * Used internally
+ */
+public class SwrveConversationConstants {
+    public static final String EXTRA_CONVERSATION_KEY = "conversation";
+    public static final String EXTRA_CONVERSATION_VERSION_KEY = "conversation_version";
```

Comment:
oops, you talking about the EXTRA_CONVERSATION_VERSION_KEY, not the EXTRA_CONVERSATION_KEY! :)
But I think its the same point.